public/app/plugins/datasource/loki/querybuilder/operationUtils.ts

```diff
@@ -148,7 +148,7 @@ function operationWithRangeVectorRenderer(
 }
 
 export function labelFilterRenderer(model: QueryBuilderOperation, def: QueryBuilderOperationDef, innerExpr: string) {
-  if (model.params[1] === '<' || model.params[1] === '>') {
+  if (model.params[1] === '<' || model.params[1] === '>' || /\d/.test(model.params[2] as string)) {
```

Comment:
First question here (specially by looking at the source of this bug): what about `>=` and `<=`?

Reply:
looking at this now, they should all be caught by `/\d/.test`.
so `model.params[1] === '<' || model.params[1] === '>'` shouldn't even be necessary now. 👀

Reply:
I would defer that to @ivanahuckova , who may have more context on why are we checking for operators here.

My suspicion was that `>, <, <=, and >=` only support numbers, while the rest threat the argument as string, but I can't back that up.